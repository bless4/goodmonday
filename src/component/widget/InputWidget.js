import React from 'react';
import { Input } from 'reactstrap';
 
export default class InputWidget extends React.Component {
    render() {
        const { props } = this;
        return(
            <Input placeholder="ex: 1,2,3,4,5,6" name={props.name} onChange={props.onChange} value={props.value} />
        );
    }
}