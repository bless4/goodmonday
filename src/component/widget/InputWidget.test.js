import React from 'react';
import { shallow } from 'enzyme';
import InputWidget from './InputWidget';

describe('InputWidget', () => {
  let component = null;
  
  it('renders correctly', () => {
    component = shallow(<InputWidget />);
  });

  it('InputWidget snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
