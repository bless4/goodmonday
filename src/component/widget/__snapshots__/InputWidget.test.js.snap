// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWidget InputWidget snapshot 1`] = `ShallowWrapper {}`;
